pkg/specgen/specgen.go

```diff
@@ -130,6 +130,10 @@ type ContainerBasicConfig struct {
 	// Remove indicates if the container should be removed once it has been started
 	// and exits
 	Remove bool `json:"remove"`
+	// PreserveFDs is a number of additional file descriptors (in addition
+	// to 0, 1, 2) that will be passed to the executed process. The total FDs
+	// passed will be 3 + PreserveFDs.
+	PreserveFDs uint `json:"-"`
```

Comment:
We do need a comment here that this is JSON:"-" because it's not available remote.

Reply:
Unlike container.go this one should stay "-" as well

Reply:
This should not be `json:"-"`

Reply:
Nevermind, I'm confused.